### PR TITLE
Ext4 Fix BlockGroup64 descriptor size to use superblock value

### PR DESCRIPTION
**Pull request by travist13**
To fix this issue: https://github.com/DiscUtils/DiscUtils/issues/130

### CODE REVIEW

File: Library/DiscUtils.Ext/VfsExtFileSystem.cs [hunk 1]
```diff
@@ -73,13 +73,13 @@ public VfsExtFileSystem(Stream stream, FileSystemParameters parameters)
             long blockDescStart = (superblock.FirstDataBlock + 1) * (long)superblock.BlockSize;
 
             stream.Position = blockDescStart;
-            var bgDescSize = superblock.Has64Bit ? BlockGroup64.DescriptorSize64 : BlockGroup.DescriptorSize;
+            var bgDescSize = superblock.Has64Bit ? superblock.DescriptorSize : BlockGroup.DescriptorSize;
```
Comment (LordMike) [inline Library/DiscUtils.Ext/VfsExtFileSystem.cs]: Could this be `var bgDescSize = superblock.DescriptorSize;`?

File: Library/DiscUtils.Ext/VfsExtFileSystem.cs [hunk 2]
```diff
@@ -73,13 +73,13 @@ public VfsExtFileSystem(Stream stream, FileSystemParameters parameters)
...
             byte[] blockDescData = StreamUtilities.ReadExact(stream, (int)numGroups * bgDescSize);
 
             _blockGroups = new BlockGroup[numGroups];
             for (int i = 0; i < numGroups; ++i)
             {
-                BlockGroup bg = superblock.Has64Bit ? new BlockGroup64() : new BlockGroup();
+                BlockGroup bg = superblock.Has64Bit ? new BlockGroup64(bgDescSize) : new BlockGroup();
```
Comment (LordMike) [inline Library/DiscUtils.Ext/VfsExtFileSystem.cs]: A bit like above, would it make sense to merge `BlockGroup` and `BlockGroup64` ?

File: Library/DiscUtils.Ext/VfsExtFileSystem.cs (hunk 1 above)
Comment (travist13) [reply Library/DiscUtils.Ext/VfsExtFileSystem.cs]: The superblock.DescriptorSize value is only valid with the SixtyFourBit feature. In my investigation, without this feature the value was often zero.

File: Library/DiscUtils.Ext/VfsExtFileSystem.cs (hunk 2 above)
Comment (travist13) [reply Library/DiscUtils.Ext/VfsExtFileSystem.cs]: Yes, I did that with the latest commit. Please take a look.

Comment (LordMike) [reply Library/DiscUtils.Ext/VfsExtFileSystem.cs]: Wow. I'm so sorry, I didn't realize BlockGroup64 inherited BlockGroup. It's best to leave them be then.

Comment (LordMike) [reply Library/DiscUtils.Ext/VfsExtFileSystem.cs]: I'm unable to revert your commit on your branch. :/

Comment (travist13) [reply Library/DiscUtils.Ext/VfsExtFileSystem.cs]: I removed the latest commit.

**Comment by travist13** [comment]
@LordMike Can I get a new release? This fix is enough for V1 of my tool. Thanks!

**Comment by LordMike** [comment]
I'll look into it. I've just noticed GitVersion added a number after Alpha .. which I'd prefer not to have.

**Pull request merged**